src/main/java/org/folio/service/inventory/InventoryItemManager.java

```diff
@@ -250,6 +249,40 @@ public Future<List<Piece>> handleItemRecords(CompositePoLine compPOL, Location l
       });
```

Comment:
handleItemRecords() does not have usages, so can be deleted

Reply:
Yes now I see it can be deleted however for `InventoryHoldingManager::getHoldingsForAllLocationTenants` it's used only in tests but we should refactor unit test and then remove it.